client/src/app/site/pages/organization/pages/designs/pages/theme-list/components/theme-list/theme-list.component.ts

```diff
@@ -40,9 +44,13 @@ export class ThemeListComponent extends BaseListViewComponent<ViewTheme> {
         }
     }
 
-    public changeCurrentTheme(theme: ViewTheme): void {
+    public async changeCurrentTheme(theme: ViewTheme): Promise<void> {
+        const title = `Are you sure you want to activate this design?`;
+        const content = `This will change the design in all committees.`;
```

Comment:
Just a spelling issue: Please use "meetings" instead of "committees" (because users working with/in meetings not in committee detail views).